### ISSUE TITLE
Container version-Cannot open /usr/output/ from marti_gui

### ISSUE THREAD

**Issue by lborcard** [richardmleggett/MARTi#4]
Hi,

I really like your app and I was trying to test it using the container with singularity. I ran into the issue that I cannot load the files to start an analysis or load an output. I am getting this error when running the container :
I also added the path of the output inside the marti_engine_options.txt as well as MARTiSampleDirectory but i am also getting this message: `MARTiSampleDirectory not specified.` 
```
Warning: Ignoring extra certs from `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`, load failed: error:02001002:system library:fopen:No such file or directory
No or invalid entry for port. Set to default (3000)

```

Could it be the cause of my problem, 

best,

Loïc

**Comment by leiste375** [comment]
Seems like it's possibly related to the issue described here:
https://github.com/npm/cli/issues/7135

Have you tried the solution suggested there?

Alternatively you can try to disable https in marti_engine_options.txt, which is fine when MARTi is only used locally.

**Comment by lborcard** [comment]
thank you for your answer, I have not tried it yet but are you certain it is related to the fact that I cannot open the /usr/reads to start an analysis?
I think https is off by default here is the beginning of the file:
```
# Default taxonomy directory can be defined here #adapted by LBO
TaxonomyDir: /usr/databases/taxonomy_6Jul20
MinKNOWRunDirectory:/usr/reads
MARTiSampleDirectory:/usr/output
Port:3000
https:false
Key:
Certificate:
```

**Comment by leiste375** [comment]
I had assumed it was related to SSL Certificates due to the warning, but I see that's likely not the case.

**Comment by lborcard** [comment]
yes I agree it's confusing, it seems that the options add by the marti_engine_options.txt are not used because I am getting the `MARTiSampleDirectory not specified.` message prompt

**Comment by nedpeel** [comment]
Hi,
I've just updated the Docker image. Do you still get the same issue using the latest?

**Comment by lborcard** [comment]
I just pulled it and it does not work either with the latest version.

**Comment by lborcard** [comment]
is there anything else that could be tried, I see that the input_folder in the GUI is called `rawDir` is it consistent with the input in the marti_engine_config.txt (`MARTiSampleDirectory `) ?

**Comment by richardmleggett** [comment]
Hi,
My colleague, Ned Peel, is looking into this. To be honest, we've never tried using the Docker image with Singularity before, but we'll look into it. If you email Ned directly (google Ned Peel Earlham Institute), it may be easier to work it through with you.
Thanks!
Richard

**Comment by lborcard** [comment]
alright, thanks a lot

**Comment by nedpeel** [comment]
Hi all,

We moved distribution of MARTi to Conda/Bioconda and are no longer publishing or maintaining a Docker image. Given we’ve deprec­ated the container path, I’m going to close this as won’t fix for Docker.

Best,
Ned
